org/lateralgm/components/ActionList.java

```diff
@@ -1015,6 +1015,7 @@ public int getSourceActions(JComponent c)
 	protected Transferable createTransferable(JComponent c)
 		{
 		indices = list.getSelectedIndices();
+		if (list.isSelectionEmpty()) return null;
```

Comment:
Does this have to be after the above line? Seems like it may as well be right before. Or you could just check that the returned list is empty. Or you can do both.

Reply:
Ok: b1c635c636097b51274f87e40349dcad6c6c85dd

No, it doesn't, good catch, I missed that.